src/command_modules/azure-cli-lab/azure/cli/command_modules/lab/sdk/devtestlabs/dev_test_labs_client.py

```diff
@@ -0,0 +1,180 @@
+# coding=utf-8
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# coding: utf-8
+# pylint: skip-file
+from msrest.service_client import ServiceClient
+from msrest import Serializer, Deserializer
+from msrestazure import AzureConfiguration
+from .version import VERSION
+from .operations.lab_operations import LabOperations
+from .operations.global_schedule_operations import GlobalScheduleOperations
+from .operations.artifact_source_operations import ArtifactSourceOperations
+from .operations.arm_template_operations import ArmTemplateOperations
+from .operations.artifact_operations import ArtifactOperations
+from .operations.cost_operations import CostOperations
+from .operations.custom_image_operations import CustomImageOperations
+from .operations.formula_operations import FormulaOperations
+from .operations.gallery_image_operations import GalleryImageOperations
+from .operations.notification_channel_operations import NotificationChannelOperations
+from .operations.policy_set_operations import PolicySetOperations
+from .operations.policy_operations import PolicyOperations
+from .operations.schedule_operations import ScheduleOperations
+from .operations.service_runner_operations import ServiceRunnerOperations
+from .operations.user_operations import UserOperations
+from .operations.disk_operations import DiskOperations
+from .operations.environment_operations import EnvironmentOperations
+from .operations.secret_operations import SecretOperations
+from .operations.virtual_machine_operations import VirtualMachineOperations
+from .operations.virtual_machine_schedule_operations import VirtualMachineScheduleOperations
+from .operations.virtual_network_operations import VirtualNetworkOperations
+from . import models
+
+
+class DevTestLabsClientConfiguration(AzureConfiguration):
+    """Configuration for DevTestLabsClient
+    Note that all parameters used to create this instance are saved as instance
+    attributes.
+
+    :param credentials: Credentials needed for the client to connect to Azure.
+    :type credentials: :mod:`A msrestazure Credentials
+     object<msrestazure.azure_active_directory>`
+    :param subscription_id: The subscription ID.
+    :type subscription_id: str
+    :param str base_url: Service URL
+    """
+
+    def __init__(
```

Comment:
that's sdk code as we are planning to ship sdk code for preview version of Azure DevTestLabs commands.